inbox/api/jsc_api.py

```diff
@@ -2,15 +2,17 @@
 import urllib
 import requests
 import simplejson
+import json
```

Comment:
IMO we can/should use [flake8](https://github.com/jobscore/sync-engine/blob/master/.travis.yml#L14) to get this information :)